Tasks/ServiceFabricDeploy/ServiceFabricSDK/Utilities.ps1

```diff
@@ -0,0 +1,207 @@
+﻿function Expand-ToFolder
+{
+    <#
+    .SYNOPSIS 
+    Unzips the zip file to the specified folder.
+
+    .PARAMETER From
+    Source location to unzip
+
+    .PARAMETER Name
+    Folder name to expand the files to.
+    #>
+
+    [CmdletBinding()]
+    Param
+    (
+        [String]
+        $File,
+        
+        [String]
+        $Destination
+    )
+
+    if (!(Test-Path $File))
+    {
+        return
+    }    
+    
+    if (Test-Path $Destination)
+    {
+        Remove-Item -Path $Destination -Recurse -ErrorAction Stop | Out-Null
+    }
+
+    New-Item $Destination -ItemType directory | Out-Null
+
+
+    Write-Verbose -Message (Get-VstsLocString -Key SFSDK_UnzipPackage -ArgumentList @($File, $Destination))
+    try 
+    {
+        [System.Reflection.Assembly]::LoadWithPartialName("System.IO.Compression.FileSystem") | Out-Null 
+        [System.IO.Compression.ZipFile]::ExtractToDirectory("$File", "$Destination") 
```

Comment:
Consider try/catch if the DLL/type can't be loaded, and fail with an actionable error message.